apps/builder/pages/api/auth/adapter.ts

```diff
@@ -42,7 +42,22 @@ export function CustomAdapter(p: PrismaClient): Adapter {
     },
     updateUser: (data) => p.user.update({ where: { id: data.id }, data }),
     deleteUser: (id) => p.user.delete({ where: { id } }),
-    linkAccount: (data) => p.account.create({ data }) as any,
+    linkAccount: (data) => {
+      return p.account.create({
+        data: {
+          provider: data.provider,
+          type: data.type,
+          providerAccountId: data.providerAccountId,
+          access_token: data.access_token,
+          token_type: data.token_type,
+          expires_at: data.expires_at,
+          refresh_token: data.refresh_token,
+          scope: data.scope,
+          id_token: data.id_token,
+          userId: data.userId,
+        }
```

Comment:
What about:

```
session_state            String?
oauth_token_secret       String?
oauth_token              String?
refresh_token_expires_in Int?
```

Reply:
Oh ok we went by the attributes that were explicitly specified for `Account`, but using the attributes from the prisma schema is definitely better.

Reply:
Should be fixed in 43bae44eb23a80f8995776fd44261de2911935d1